tensortrade/trades/trade.py

```diff
@@ -11,63 +11,149 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-
+import uuid
 
 class Trade(object):
```

Comment:
Seems to be missing the new `step: int` property to track the `current_step` that the trade was made on.